src/syft/core/node/device/device.py

```diff
@@ -52,6 +52,9 @@ def __init__(
             verify_key=verify_key,
         )
 
+        # specific location with name
```

Comment:
not sure i like the name of the attribute here - in the long run there should be a set of locations but it doesn't have to be that way for now

Reply:
I guess that means we should be thinking more about the switch to has-a rather than is-a for Node / Client <- Address.